labelmerge/workflow/scripts/labelmerge.py

```diff
@@ -151,21 +175,28 @@ def main():
     overlay_exceptions = (
         args.overlay_exceptions if args.overlay_exceptions else []
     )
+    base_drops = args.base_drops if args.base_drops else []
+    overlay_drops = args.overlay_drops if args.overlay_drops else []
     base_datasets = split_labels(
         base_data,
         base_metadata,
         prefix="base ",
         exceptions=base_exceptions,
+        drops=base_drops,
     )
     overlay_datasets = split_labels(
         overlay_data,
         overlay_metadata,
         prefix="overlay ",
         exceptions=overlay_exceptions,
+        drops=overlay_drops,
     )
     # Note that overlay exceptions are ignored
     merged_map, merged_metadata = merge_labels(
```

Comment:
Could you just walk me through this? From what I gather, we are adding (in order): 

overlay exceptions -> base inclusions -> overlay inclusions -> base exceptions?

I am guessing this results in some overwriting of the existing labels?

Reply:
Sure:
`split_labels` considers each unique label found in a label map. If that label is in the list of `drops`, it is ignored. If that label is in the list of `exclusions`, it's split into a dataset that contains all the exclusion masks. Otherwise, it's split into a dataset that contains all the "regular" masks. (If there are no exceptions, only the dataset of "regular" masks is returned)

So, after applying `split_labels` to the base and overlay input label maps, we have a base dataset, an overlay dataset, and maybe a base exclusions dataset and/or overlay exclusion dataset.

We then pass a list of those datasets in the priority order you identified (overlay exceptions (if applicable) -> base regular -> overlay regular -> base exceptions (if applicable)) to `merge_labels`, which reindexes all the label names that show up in the datasets, and writes the masks to an output 3D array with the new indices in order, so that later datasets overwrite labels written by earlier datasets.

Reply:
Sounds good - I think there is still the fundamental concern about mixed labels for voxels that don't overlap, but that was present prior to this. In any case, this PR looks good to merge to me.